packages/core/README.md

```diff
@@ -69,11 +80,10 @@ The mapConfiguration allows controlling many client instance details.
 
 | fieldName | type | description |
 | - | - | - |
-| language | enum["de", "en"] | Initial language. |
-| layerConf | LayerConf | Layer configuration as required by masterportalAPI. |
-| <...masterportalAPI.fields> | various | The object is also used to initialize the masterportalAPI. Please refer to their documentation for options. |
+| <...masterportalapi.fields> | various | Multiple different parameters are required by the masterportalapi to be able to create the map. Also, some fields are optional but relevant and thus described here as well. For all additional options, refer to the documentation of the masterportalapi itself. |
 | checkServiceAvailability | boolean? | If set to `true`, all services' availability will be checked with head requests. |
-| extendedMasterportalapiMarkers | extendedMasterportalapiMarkers? | Optional. If set, all configured visible vector layers' features can be hovered and selected by mouseover and click respectively. They are available as features in the store. Layers with `clusterDistance` will be clustered to a multi-marker that supports the same features. Please mind that this only works properly if you configure nothing but point marker vector layers styled by the masterportalAPI. |
+| extendedMasterportalapiMarkers | extendedMasterportalapiMarkers? | Optional. If set, all configured visible vector layers' features can be hovered and selected by mouseover and click respectively. They are available as features in the store. Layers with `clusterDistance` will be clustered to a multi-marker that supports the same features. Please mind that this only works properly if you configure nothing but point marker vector layers styled by the masterportalapi. |
+| language | enum["de", "en"]? | Initial language. |
```

Comment:
Why has this become optional? What is the default value? 🤔

Reply:
This has always been optional, the documentation was just wrong